backend/corpora/api_server/app.py

```diff
@@ -47,6 +48,10 @@ def add_api(base_path, spec_file):
     add_api(base_path="/wmg", spec_file="wmg-api.yml")
     add_api(base_path="/gene_info", spec_file="gene-info-api.yml")
 
+    # initialize gene checker to go ahead and create a dictionary of all
```

Comment:
can you also add a comment about why this is being done here (instead of on request) so future engineers arent confused